examples/tutorials/translations/bengali/Part 03 - Advanced Remote Execution Tools.ipynb

```diff
@@ -0,0 +1,402 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Part 3: উন্নত রিমোট এক্সিকিউশন সরঞ্জাম (Advanced Remote Execution Tools)\n",
+    "\n",
+    "শেষ বিভাগে আমরা ফেডারেটেড লার্নিং ব্যবহার করে একটি খেলনা মডেল প্রশিক্ষণ দিয়েছি। আমরা এটি কল করেই করেছি .send() এবং .get() আমাদের মডেল উপর এটিকে প্রশিক্ষণ ডেটার অবস্থানটিতে প্রেরণ করা, এটি আপডেট করা এবং তারপরে এনে ফিরিয়ে আনা। তবে উদাহরণটির শেষে আমরা বুঝতে পেরেছিলাম যে মানুষের গোপনীয়তা রক্ষা করতে আমাদের আরও কিছুটা এগিয়ে যাওয়া দরকার. যথা, আমরা গ্রেডিয়েন্টগুলি গড় করতে চাই **আগে** কলিং .get(). এইভাবে, আমরা কখনই কারওর সঠিক গ্রেডিয়েন্টটি দেখতে পাব না (এভাবে তাদের গোপনীয়তা রক্ষা করা আরও ভাল !!!)\n",
```

Comment:
"শেষ বিভাগে আমরা ফেডারেটেড লার্নিং ব্যবহার করে একটি খেলনা মডেল প্রশিক্ষণ দিয়েছি। আমরা এটি করেছি আমাদের মডেলের উপর .send() এবং .get() কল করে,  এটিকে প্রশিক্ষণ ডেটার অবস্থানটিতে প্রেরণ করে, এটি আপডেট করে এবং তারপরে  ফিরিয়ে আনা। তবে উদাহরণটির শেষে আমরা বুঝতে পেরেছিলাম যে মানুষের গোপনীয়তা রক্ষা করতে আমাদের আরও কিছুটা এগিয়ে যাওয়া দরকার। যথা- আমরা .get() কল করার পূর্বে গ্রেডিয়েন্টগুলি গড় করতে চাই। এইভাবে, আমরা কখনই কারোর সঠিক গ্রেডিয়েন্টটি দেখতে পাবো না (এভাবে তাদের গোপনীয়তা রক্ষা করা আরও ভাল !!!)\n"

Reply:
Yes, It will be easy to read! made changes!